sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql

```diff
@@ -7,3 +7,26 @@ UPDATE Study SET studyTypeId = (SELECT typeId FROM StudyType WHERE name = studyT
 ALTER TABLE Study ADD CONSTRAINT study_studyTypeId FOREIGN KEY (studyTypeId) REFERENCES StudyType(typeId);
 ALTER TABLE Study DROP COLUMN studyType;
 ALTER TABLE Study ALTER COLUMN studyTypeId bigint(20) NOT NULL;
+
+ALTER TABLE Status ADD FOREIGN KEY (instrumentName) REFERENCES SequencerReference (name);
+ALTER TABLE Status ADD FOREIGN KEY (runName) REFERENCES Run (alias);
```

Comment:
Not without refactoring the notification servers -- there are several instances of  `requestManager.getSequencerReferenceByName(r.getStatus().getInstrumentName());`